src/ui/locales/id.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Alihkan atribusi",
+    "AttributionControl.MapFeedback": "Umpan balik peta",
+    "FullscreenControl.Enter": "Memasuki layar penuh",
+    "FullscreenControl.Exit": "Keluar dari layar penuh",
+    "GeolocateControl.FindMyLocation": "Temukan lokasi saya",
+    "GeolocateControl.LocationNotAvailable": "Lokasi tidak tersedia",
+    "LogoControl.Title": "logo kotak peta",
+    "Map.Title": "Peta",
+    "NavigationControl.ResetBearing": "Setel ulang bantalan ke utara",
```

Comment:
Bearing synonym is azimuth [(wikipedia)](https://en.wikipedia.org/wiki/Bearing_(angle)). The translation of azimuth in Indonesian is `azimut`.
```suggestion
    "NavigationControl.ResetBearing": "Setel ulang azimut ke utara",
```